tests/main/parallel-install-services/task.yaml

```diff
@@ -45,6 +45,36 @@ execute: |
     snap list | MATCH '^test-snapd-service_foo '
     snap list | MATCH '^test-snapd-service_longname '
 
+    echo "Ensure we can use snapctl to stop a service the main installation, but not affect others"
+    snap run --shell test-snapd-service -c "snapctl stop test-snapd-service.test-snapd-other-service"
+    systemctl is-active snap.test-snapd-service.test-snapd-other-service | MATCH "inactive"
+    systemctl is-active snap.test-snapd-service_foo.test-snapd-other-service | MATCH "^active"
+    systemctl is-active snap.test-snapd-service_longname.test-snapd-other-service | MATCH "^active"
+
+    echo "Ensure we can start it again"
+    snap run --shell test-snapd-service -c "snapctl start test-snapd-service.test-snapd-other-service"
+    systemctl is-active snap.test-snapd-service.test-snapd-other-service | MATCH "^active"
+    systemctl is-active snap.test-snapd-service_foo.test-snapd-other-service | MATCH "^active"
+    systemctl is-active snap.test-snapd-service_longname.test-snapd-other-service | MATCH "^active"
+
+    echo "Ensure we can use snapctl to stop a service in parallel installs without affecting others"
+    for instance in foo longname; do
+        echo "Testing stop of test-snapd-service_$instance"
+        systemctl is-active "snap.test-snapd-service_${instance}.test-snapd-other-service" | MATCH "^active"
+        snap run --shell test-snapd-service_${instance} -c "snapctl stop test-snapd-service_${instance}.test-snapd-other-service"
+        systemctl is-active "snap.test-snapd-service_${instance}.test-snapd-other-service" | MATCH "inactive"
+    done
+
+    echo "Verifying the main install should be unaffected after stop"
+    systemctl is-active snap.test-snapd-service.test-snapd-other-service | MATCH "^active"
```

Comment:
Can you also add verification of the inverse case when snap_instance tries to manage non suffixed snap services?

```
echo "Verifying snap can only manage its own services"
snap run --shell test-snapd-service_foo -c "snapctl stop test-snapd-service.test-snapd-other-service" |& MATCH "error: blah blah"
```